docs/core_docs/docs/integrations/chat/friendli.mdx

```diff
@@ -0,0 +1,29 @@
+# Friendli
+
+> [Friendli](https://friendli.ai/) enhances AI application performance and optimizes cost savings with scalable, efficient deployment options, tailored for high-demand AI workloads.
+
+This tutorial guides you through integrating `ChatFriendli` for chat applications using LangChain. `ChatFriendli` offers a flexible approach to generating conversational AI responses, supporting both synchronous and asynchronous calls.
+
+## Setup
+
+Ensure the `@langchain/community` is installed.
+
+import IntegrationInstallTooltip from "@mdx_components/integration_install_tooltip.mdx";
+
+<IntegrationInstallTooltip></IntegrationInstallTooltip>
+
+```bash npm2yarn
+npm install replicate @langchain/community
+```
+
+Sign in to [Friendli Suite](https://suite.friendli.ai/) to create a Personal Access Token, and set it as the `FRIENDLI_TOKEN` environment.
+You can set team id as `FRIENDLI_TEAM` environment.
+
+You can initialize a Friendli chat model with selecting the model you want to use. The default model is `mixtral-8x7b-instruct-v0-1`. You can check the available models at [docs.friendli.ai](https://docs.periflow.ai/guides/serverless_endpoints/pricing#text-generation-models).
```

Comment:
change to docs.friendli.ai

Reply:
@kooyunmo Please check if the python langchain version uses the periflow.ai link as well

Reply:
@kooyunmo There is typos that I found from https://github.com/langchain-ai/langchain/pull/17913/files.
- `docs/docs/integrations/chat/friendli.ipynb` line 74, 'FrienliChat'
- `docs/docs/integrations/llms/friendli.ipynb` line 74, 'Frienli'
- `docs/docs/integrations/chat/friendli.ipynb` line 54, 'default model is mixtral-8x7b-instruct-v0-1...' (but default is llama-2-13b-chat.)
- `docs/docs/integrations/chat/friendli.ipynb` line 54, 'docs.periflow.ai'
- `docs/docs/integrations/llms/friendli.ipynb` line 54, 'docs.periflow.ai'

Reply:
@kooyunmo

Reply:
@kooyunmo 
- `docs/docs/integrations/chat/friendli.ipynb` line 47, 'Friendi Personal Access Token'
- `docs/docs/integrations/llms/friendli.ipynb` line 47, 'Friendi Personal Access Token'